web/src/components/Plugin/UI/basic-auth.tsx

```diff
@@ -20,6 +20,7 @@ import { Form, Input } from 'antd';
 
 type Props = {
   form: FormInstance;
+  schema: Record<string, any> | undefined;
```

Comment:
Hi, @wmdmomo I couldn't find out where this field is used, could you please have a better description?

Reply:
![1627214218(1)](https://user-images.githubusercontent.com/43945660/126898129-db5ab8a0-fd04-461d-a114-ef7323ef200f.png)
Because for the plug-in basic-auth, its back-end jsonschema is not specifically used, so here is just passing parameters.

Reply:
Hi @WillemJiang , sorry for taking so long to reply.
In fact there are two schema for `basic-auth` plugin, one is the `schema` as you pointed in the Screenshot, this is used for `Route` or `Service`. The other is the `consumer_schema`, this is for `Consumer`. 

![image](https://user-images.githubusercontent.com/2561857/130165910-cbef8da2-e40f-4e4d-b273-627ba5b4b894.png)

`basic-auth` form UI is used for `Consumer`, so please use the `consumer_schema`.

Reply:
@liuxiran Thank you for your reminder. I have modified it. When the plugin is basic-auth, pass it as the value of consumer_schema.

Reply:
nice, and then @wmdmomo  can try to dynamically adapt to the required props. which  the only rule for `basic-auth` plugin

Reply:
@liuxiran  Sorry for taking so long to reply. I have now verified the required items of the basic-auth plugin.